sdk/src/main/scala/com/horizen/fork/SidechainFork1.scala

```diff
@@ -1,5 +1,5 @@
 package com.horizen.fork
 
 class SidechainFork1(override val epochNumber: ForkConsensusEpochNumber) extends BaseConsensusEpochFork(epochNumber) {
```

Comment:
Let's add more description before the class, explaining the purpose and the changes of the fork. Can be very useful for future.

Reply:
Sure, should i add also the info related to the openStakeTransaction?

Reply:
yes, please

Reply:
Done